### PR TITLE
schema update changes

### PR DESCRIPTION
**Pull request by anshulforyou**

### CODE REVIEW

File: airstack-modules/modules/airstack/utils.ts [hunk 1]
```diff
@@ -2,10 +2,14 @@ import {
...
 export const BIGINT_ONE = BigInt.fromI32(1);
-export const SUBGRAPH_VERSION = "1";
+export const SUBGRAPH_VERSION = "1.0.0";
 export const SUBGRAPH_NAME = "AIRSTACK-SUBGRAPH";
-export const SUBGRAPH_SLUG = "AIRSTACK-SUBGRAPH";
\ No newline at end of file
+export const SUBGRAPH_SLUG = "AIRSTACK-SUBGRAPH";
+
+export function processNetwork(network: string): string{
+    return network.trim().toUpperCase().replace("-", "_");
```
Comment (0xsarvesh) [inline airstack-modules/modules/airstack/utils.ts:14]: Instead of this create a mapping.

**Review by 0xsarvesh — CHANGES_REQUESTED** [review]

File: airstack-modules/src/integrate.ts [hunk 2]
```diff
@@ -244,6 +244,6 @@ function getAirMetaDetails(){
   let fileContent = fs.readFileSync(targetFile, { encoding: "utf8" });
   fileContent = fileContent.replace(/export const SUBGRAPH_NAME = ".*";/g, "");
   fileContent = fileContent.replace(/export const SUBGRAPH_SLUG = ".*";/g, "");
-  fileContent += `export const SUBGRAPH_NAME = "${SUBGRAPH_NAME}";\nexport const SUBGRAPH_SLUG = "${SUBGRAPH_SLUG}";\n`
+  fileContent += `\nexport const SUBGRAPH_NAME = "${SUBGRAPH_NAME}";\nexport const SUBGRAPH_SLUG = "${SUBGRAPH_SLUG}";\n`
```
[the remark this answers is not in the document]
Comment (anshulforyou) [reply airstack-modules/src/integrate.ts:247]: No, it's an improvement. SUBGRAPH_NAME and SLUG will now be added to new line, previously they were being added to the last line

**Review by 0xsarvesh — DISMISSED** [review]

File: airstack-modules/graphql/airstack-nft-marketplace-schema.graphql.ts [hunk 3]
```diff
@@ -104,34 +98,28 @@ type AirAccount @entity {
...
-  transactions: [AirNftTransaction!] @derivedFrom(field: "transactionToken")
 }
 
-
 interface AirTransaction {
 	id: ID ! #chain-hash-logIndex-< add as per context > NFT marketplace will have tokenId
   from: AirAccount!
   to: AirAccount!
-  hash: String!
+  hash: String! #todo: Check if this should be bytes
```
Comment (0xsarvesh) [inline airstack-modules/graphql/airstack-nft-marketplace-schema.graphql.ts:110]: Can you remove this comments.

Comment (anshulforyou) [reply airstack-modules/graphql/airstack-nft-marketplace-schema.graphql.ts]: Done

**Review by 0xsarvesh — APPROVED** [review]

**Pull request merged**